Build failure: punes

### Nixpkgs version

- Unstable (25.11)

### Steps to reproduce

- be on rev:c5ae371f1a6a7fd27823bc500d9390b38c05fa55
- nix build nixpkgs#punes

### Can Hydra reproduce this build failure?

Yes, Hydra can reproduce this build failure.

### Link to Hydra build job

https://hydra.nixos.org/build/313212985

### Relevant log output

```console
Running phase: qtPreHook
Running phase: unpackPhase
unpacking source archive /nix/store/36cm0hg48bjg9kypcf5mllk4svn3n564-source
source root is source
Running phase: patchPhase
applying patch /nix/store/zswrk4r9gpi61y218vjpv64qpgq9ywxa-0001-punes-Fix-compatibility-with-Qt-6.7.1.patch
patching file src/gui/dlgStdPad.hpp
applying patch /nix/store/d0v3y8i0nii8bjlhv2hisdzj3hhnxkfw-0002-punes-Updated-code-for-Qt-6.9.0-compatibility.patch
patching file src/gui/qt.cpp
Hunk #1 succeeded at 930 (offset -207 lines).
Hunk #2 succeeded at 941 (offset -207 lines).
Hunk #3 succeeded at 950 (offset -207 lines).
patching file src/gui/wdgStatusBar.cpp
Hunk #1 succeeded at 139 (offset 2 lines).
Running phase: updateAutotoolsGnuConfigScriptsPhase
Running phase: configurePhase
cmake flags: -DCMAKE_FIND_USE_SYSTEM_PACKAGE_REGISTRY=OFF -DCMAKE_FIND_USE_PACKAGE_REGISTRY=OFF -DCMAKE_EXPORT_NO_PACKAGE_REGISTRY=ON -DCMAKE_BUILD_TYPE=Release -DBUILD_TESTING=OFF -DCMAKE_INSTALL_LOCALEDIR=/nix/store/hd0q300wnmvf0wk3hkjvjv6akajrvbi7-punes-0.111/share/locale -DCMAKE_INSTALL_LIBEXECDIR=/nix/store/hd0q300wnmvf0wk3hkjvjv6akajrvbi7-punes-0.111/libexec -DCMAKE_INSTALL_LIBDIR=/nix/store/hd0q300wnmvf0wk3hkjvjv6akajrvbi7-punes-0.111/lib -DCMAKE_INSTALL_DOCDIR=/nix/store/hd0q300wnmvf0wk3hkjvjv6akajrvbi7-punes-0.111/share/doc/puNES -DCMAKE_INSTALL_INFODIR=/nix/store/hd0q300wnmvf0wk3hkjvjv6akajrvbi7-punes-0.111/share/info -DCMAKE_INSTALL_MANDIR=/nix/store/hd0q300wnmvf0wk3hkjvjv6akajrvbi7-punes-0.111/share/man -DCMAKE_INSTALL_INCLUDEDIR=/nix/store/hd0q300wnmvf0wk3hkjvjv6akajrvbi7-punes-0.111/include -DCMAKE_INSTALL_SBINDIR=/nix/store/hd0q300wnmvf0wk3hkjvjv6akajrvbi7-punes-0.111/sbin -DCMAKE_INSTALL_BINDIR=/nix/store/hd0q300wnmvf0wk3hkjvjv6akajrvbi7-punes-0.111/bin -DCMAKE_INSTALL_NAME_DIR=/nix/store/hd0q300wnmvf0wk3hkjvjv6akajrvbi7-punes-0.111/lib -DCMAKE_STRIP=/nix/store/ag7fc58qzff4j1sgd7v98irhcr6vcg8v-gcc-wrapper-14.3.0/bin/strip -DCMAKE_RANLIB=/nix/store/ag7fc58qzff4j1sgd7v98irhcr6vcg8v-gcc-wrapper-14.3.0/bin/ranlib -DCMAKE_AR=/nix/store/ag7fc58qzff4j1sgd7v98irhcr6vcg8v-gcc-wrapper-14.3.0/bin/ar -DCMAKE_C_COMPILER=gcc -DCMAKE_CXX_COMPILER=g++ -DCMAKE_INSTALL_PREFIX=/nix/store/hd0q300wnmvf0wk3hkjvjv6akajrvbi7-punes-0.111 -DENABLE_GIT_INFO=OFF -DENABLE_RELEASE=ON -DENABLE_FFMPEG=ON -DENABLE_OPENGL=ON -DENABLE_QT6_LIBS:BOOL=FALSE
-- The C compiler identification is GNU 14.3.0
-- The CXX compiler identification is GNU 14.3.0
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working C compiler: /nix/store/ag7fc58qzff4j1sgd7v98irhcr6vcg8v-gcc-wrapper-14.3.0/bin/gcc - skipped
-- Detecting C compile features
-- Detecting C compile features - done
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: /nix/store/ag7fc58qzff4j1sgd7v98irhcr6vcg8v-gcc-wrapper-14.3.0/bin/g++ - skipped
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Looking for sys/types.h
-- Looking for sys/types.h - found
-- Looking for stdint.h
-- Looking for stdint.h - found
-- Looking for stddef.h
-- Looking for stddef.h - found
-- Check size of size_t
-- Check size of size_t - done
-- Check size of unsigned int
-- Check size of unsigned int - done
-- Check size of unsigned long
-- Check size of unsigned long - done
-- Check size of unsigned long long
-- Check size of unsigned long long - done
-- Found ALSA: /nix/store/m9wcg7lpb9i43yrjyk0k8kcf3pgicn8s-alsa-lib-1.2.14/lib/libasound.so (found version "1.2.14")
-- Found PkgConfig: /nix/store/jdphp8w273n02mxi9gdpdyy0qqsha70r-pkg-config-wrapper-0.29.2/bin/pkg-config (found version "0.29.2")
-- Checking for module 'libudev'
--   Found libudev, version 258
Package libcap was not found in the pkg-config search path.
Perhaps you should add the directory containing `libcap.pc'
to the PKG_CONFIG_PATH environment variable
Package 'libcap', required by 'libudev', not found
Package libcap was not found in the pkg-config search path.
Perhaps you should add the directory containing `libcap.pc'
to the PKG_CONFIG_PATH environment variable
Package 'libcap', required by 'libudev', not found
Package libcap was not found in the pkg-config search path.
Perhaps you should add the directory containing `libcap.pc'
to the PKG_CONFIG_PATH environment variable
Package 'libcap', required by 'libudev', not found
Package libcap was not found in the pkg-config search path.
Perhaps you should add the directory containing `libcap.pc'
to the PKG_CONFIG_PATH environment variable
Package 'libcap', required by 'libudev', not found
-- Found X11: /nix/store/fz1llaam5z4ix2akx2ix7gjs9lqy21ds-xorgproto-2024.1/include
-- Looking for XOpenDisplay in /nix/store/rg7xk1jnf4i5sqn55s8x846mgp656lg9-libx11-1.8.12/lib/libX11.so;/nix/store/jh67r1va6azmv0pi7j1xdjk5ffgw6yw0-libxext-1.3.6/lib/libXext.so
-- Looking for XOpenDisplay in /nix/store/rg7xk1jnf4i5sqn55s8x846mgp656lg9-libx11-1.8.12/lib/libX11.so;/nix/store/jh67r1va6azmv0pi7j1xdjk5ffgw6yw0-libxext-1.3.6/lib/libXext.so - found
-- Looking for gethostbyname
-- Looking for gethostbyname - found
-- Looking for connect
-- Looking for connect - found
-- Looking for remove
-- Looking for remove - found
-- Looking for shmat
-- Looking for shmat - found
-- Found OpenGL: /nix/store/lgrb82lslvxgcv01dvxxqx6cjc3l3qdy-libglvnd-1.7.0/lib/libOpenGL.so
-- Found Qt: 5.15.18
-- Performing Test CMAKE_HAVE_LIBC_PTHREAD
-- Performing Test CMAKE_HAVE_LIBC_PTHREAD - Success
-- Found Threads: TRUE
-- Checking for modules 'libavformat;libavcodec;libswresample;libswscale;libavutil'
--   Found libavformat, version 62.3.100
--   Found libavcodec, version 62.11.100
--   Found libswresample, version 6.1.100
--   Found libswscale, version 9.1.100
--   Found libavutil, version 60.8.100
-- Configuring done (10.0s)
-- Generating done (0.7s)
CMake Warning:
  Manually-specified variables were not used by the project:

    BUILD_TESTING
    CMAKE_EXPORT_NO_PACKAGE_REGISTRY


-- Build files have been written to: /build/source/build
cmake: enabled parallel building
cmake: enabled parallel installing
Running phase: buildPhase
build flags: -j24 SHELL=/nix/store/qsydfxm1vq6q9jac2kq3r8kn0xdmsldf-bash-5.3p3/bin/bash
[  0%] Built target lib7zip_autogen_timestamp_deps
[  0%] Built target qkeycode_autogen_timestamp_deps
[  0%] Automatic MOC and UIC for target lib7zip
[  0%] Automatic MOC and UIC for target qkeycode
[  0%] Built target qkeycode_autogen
[  0%] Built target lib7zip_autogen
[  0%] Building CXX object _deps/qkeycode-build/CMakeFiles/qkeycode.dir/qkeycode_autogen/mocs_compilation.cpp.o
[  1%] Building CXX object _deps/qkeycode-build/CMakeFiles/qkeycode.dir/src/qkeycode/chromium/keycode_converter.cc.o
[  1%] Building CXX object _deps/qkeycode-build/CMakeFiles/qkeycode.dir/src/qkeycode/qkeycode.cpp.o
[  1%] Building CXX object _deps/lib7zip-build/src/CMakeFiles/lib7zip.dir/lib7zip_autogen/mocs_compilation.cpp.o
[  1%] Building CXX object _deps/lib7zip-build/src/CMakeFiles/lib7zip.dir/7ZipArchive.cpp.o
[  1%] Building CXX object _deps/lib7zip-build/src/CMakeFiles/lib7zip.dir/7ZipCompressCodecsInfo.cpp.o
[  2%] Building CXX object _deps/lib7zip-build/src/CMakeFiles/lib7zip.dir/7ZipInStreamWrapper.cpp.o
[  2%] Building CXX object _deps/lib7zip-build/src/CMakeFiles/lib7zip.dir/OSFunctions_UnixLike.cpp.o
[  2%] Building CXX object _deps/lib7zip-build/src/CMakeFiles/lib7zip.dir/HelperFuncs.cpp.o
[  2%] Building CXX object _deps/lib7zip-build/src/CMakeFiles/lib7zip.dir/7ZipArchiveOpenCallback.cpp.o
[  2%] Building CXX object _deps/lib7zip-build/src/CMakeFiles/lib7zip.dir/7ZipArchiveItem.cpp.o
[  2%] Building CXX object _deps/lib7zip-build/src/CMakeFiles/lib7zip.dir/7ZipObjectPtrArray.cpp.o
[  2%] Building CXX object _deps/lib7zip-build/src/CMakeFiles/lib7zip.dir/OSFunctions_Win32.cpp.o
[  2%] Building CXX object _deps/lib7zip-build/src/CMakeFiles/lib7zip.dir/7ZipFormatInfo.cpp.o
[  2%] Building CXX object _deps/lib7zip-build/src/CMakeFiles/lib7zip.dir/7ZipDllHandler.cpp.o
[  3%] Building CXX object _deps/lib7zip-build/src/CMakeFiles/lib7zip.dir/7ZipCodecInfo.cpp.o
[  3%] Building CXX object _deps/lib7zip-build/src/CMakeFiles/lib7zip.dir/7ZipOpenArchive.cpp.o
[  3%] Building CXX object _deps/lib7zip-build/src/CMakeFiles/lib7zip.dir/7zipLibrary.cpp.o
[  3%] Building CXX object _deps/lib7zip-build/src/CMakeFiles/lib7zip.dir/OSFunctions_OS2.cpp.o
[  3%] Building CXX object _deps/lib7zip-build/src/CMakeFiles/lib7zip.dir/__/__/p7zip-17.04/CPP/Common/MyWindows.cpp.o
[  3%] Building CXX object _deps/lib7zip-build/src/CMakeFiles/lib7zip.dir/__/__/p7zip-17.04/CPP/Windows/PropVariant.cpp.o
[  4%] Linking CXX static library lib7zip.a
[  4%] Built target lib7zip
[  4%] Linking CXX static library libqkeycode.a
[  4%] Built target qkeycode
[  4%] Built target punes_autogen_timestamp_deps
[  4%] Automatic MOC and UIC for target punes
[  4%] Built target punes_autogen
[  4%] Generating translations/tr_TR.qm
[  4%] Generating translations/qt_pl.qm
[  4%] Generating translations/zh_CN.qm
[  4%] Generating translations/qt_pt_BR.qm
[  4%] Generating translations/qt_tr.qm
[  4%] Generating translations/qt_es.qm
[  5%] Generating translations/qt_it.qm
[  5%] Generating translations/fr_FR.qm
[  6%] Generating translations/de_DE.qm
[  6%] Generating translations/en_EN.qm
[  6%] Generating translations/qt_hu.qm
[  7%] Generating translations/ru_RU.qm
Updating '/build/source/build/src/translations/zh_CN.qm'...
Updating '/build/source/build/src/translations/fr_FR.qm'...
    Generated 941 translation(s) (936 finished and 5 unfinished)
Updating '/build/source/build/src/translations/tr_TR.qm'...
    Generated 941 translation(s) (941 finished and 0 unfinished)
Updating '/build/source/build/src/translations/qt_tr.qm'...
[  7%] Generating translations/it_IT.qm
Updating '/build/source/build/src/translations/en_EN.qm'...
    Generated 941 translation(s) (739 finished and 202 unfinished)
    Generated 0 translation(s) (0 finished and 0 unfinished)
    Ignored 941 untranslated source text(s)
Updating '/build/source/build/src/translations/qt_es.qm'...
Updating '/build/source/build/src/translations/ru_RU.qm'...
[  7%] Generating translations/ar_AR.qm
Updating '/build/source/build/src/translations/qt_pl.qm'...
Updating '/build/source/build/src/translations/de_DE.qm'...
[  7%] Generating translations/es_ES.qm
    Generated 1786 translation(s) (1785 finished and 1 unfinished)
    Generated 941 translation(s) (941 finished and 0 unfinished)
[  7%] Generating translations/qt_ru.qm
[  7%] Generating translations/hu_HU.qm
[  7%] Generating translations/pt_BR.qm
    Generated 1519 translation(s) (1513 finished and 6 unfinished)
    Ignored 267 untranslated source text(s)
    Generated 941 translation(s) (613 finished and 328 unfinished)
Updating '/build/source/build/src/translations/qt_pt_BR.qm'...
    Generated 1462 translation(s) (1455 finished and 7 unfinished)
    Ignored 324 untranslated source text(s)
[  8%] Generating translations/pl_PL.qm
Updating '/build/source/build/src/translations/qt_it.qm'...
[  8%] Generating translations/qt_ar.qm
[  8%] Generating translations/qt_de.qm
Updating '/build/source/build/src/translations/qt_hu.qm'...
    Generated 1455 translation(s) (1447 finished and 8 unfinished)
    Ignored 331 untranslated source text(s)
    Generated 1786 translation(s) (1786 finished and 0 unfinished)
    Generated 1463 translation(s) (1456 finished and 7 unfinished)
    Ignored 323 untranslated source text(s)
Updating '/build/source/build/src/translations/hu_HU.qm'...
[  8%] Generating translations/qt_zh_CN.qm
[  8%] Generating translations/qt_fr.qm
[  8%] Automatic RCC for gui/designer/resources.qrc
Updating '/build/source/build/src/translations/ar_AR.qm'...
Updating '/build/source/build/src/translations/it_IT.qm'...
Updating '/build/source/build/src/translations/es_ES.qm'...
    Generated 941 translation(s) (941 finished and 0 unfinished)
Updating '/build/source/build/src/translations/pt_BR.qm'...
    Generated 941 translation(s) (941 finished and 0 unfinished)
    Generated 941 translation(s) (729 finished and 212 unfinished)
    Generated 941 translation(s) (939 finished and 2 unfinished)
Updating '/build/source/build/src/translations/pl_PL.qm'...
Updating '/build/source/build/src/translations/qt_ru.qm'...
    Generated 941 translation(s) (941 finished and 0 unfinished)
    Generated 941 translation(s) (926 finished and 15 unfinished)
    Generated 1786 translation(s) (1784 finished and 2 unfinished)
Updating '/build/source/build/src/translations/qt_fr.qm'...
Updating '/build/source/build/src/translations/qt_de.qm'...
Updating '/build/source/build/src/translations/qt_ar.qm'...
    Generated 1452 translation(s) (1443 finished and 9 unfinished)
    Ignored 334 untranslated source text(s)
    Generated 1786 translation(s) (1785 finished and 1 unfinished)
    Generated 1574 translation(s) (1568 finished and 6 unfinished)
    Ignored 212 untranslated source text(s)
Updating '/build/source/build/src/translations/qt_zh_CN.qm'...
    Generated 1696 translation(s) (1696 finished and 0 unfinished)
    Ignored 90 untranslated source text(s)
[  8%] Automatic RCC for translations.qrc
[  8%] Building C object src/CMakeFiles/punes.dir/audio/blipbuf.c.o
[  8%] Building C object src/CMakeFiles/punes.dir/audio/channels.c.o
[  8%] Building CXX object src/CMakeFiles/punes.dir/punes_autogen/mocs_compilation.cpp.o
[  8%] Building C object src/CMakeFiles/punes.dir/audio/handler.c.o
[  8%] Building C object src/CMakeFiles/punes.dir/audio/blip_buf.c.o
[  9%] Building C object src/CMakeFiles/punes.dir/audio/delay.c.o
[  9%] Building C object src/CMakeFiles/punes.dir/audio/panning.c.o
[  9%] Building C object src/CMakeFiles/punes.dir/audio/mono.c.o
[  9%] Building C object src/CMakeFiles/punes.dir/audio/wave.c.o
[ 10%] Building C object src/CMakeFiles/punes.dir/core/input/arkanoid.c.o
[ 10%] Building C object src/CMakeFiles/punes.dir/audio/alsa/snd.c.o
[ 10%] Building C object src/CMakeFiles/punes.dir/core/input/famicom.c.o
[ 10%] Building C object src/CMakeFiles/punes.dir/core/input/nes_001.c.o
[ 10%] Building C object src/CMakeFiles/punes.dir/core/input/family_basic_keyboard.c.o
[ 10%] Building C object src/CMakeFiles/punes.dir/core/input/four_score.c.o
[ 10%] Building C object src/CMakeFiles/punes.dir/core/input/nsf_controller.c.o
[ 11%] Building C object src/CMakeFiles/punes.dir/core/input/nsf_mouse.c.o
[ 11%] Building C object src/CMakeFiles/punes.dir/core/input/vs.c.o
[ 11%] Building C object src/CMakeFiles/punes.dir/core/input/snes_mouse.c.o
[ 12%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_000.c.o
[ 12%] Building C object src/CMakeFiles/punes.dir/core/input/zapper.c.o
[ 12%] Building C object src/CMakeFiles/punes.dir/core/input/oeka_kids_tablet.c.o
[ 12%] Building C object src/CMakeFiles/punes.dir/core/input/standard_controller.c.o
[ 12%] Building C object src/CMakeFiles/punes.dir/core/input/subor_keyboard.c.o
[ 12%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_001.c.o
[ 12%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_002.c.o
[ 12%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_003.c.o
[ 12%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_005.c.o
[ 12%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_004.c.o
[ 12%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_007.c.o
[ 13%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_006.c.o
[ 13%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_009.c.o
[ 13%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_010.c.o
[ 13%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_013.c.o
[ 13%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_011.c.o
[ 13%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_012.c.o
[ 14%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_014.c.o
[ 14%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_015.c.o
[ 14%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_016.c.o
[ 14%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_018.c.o
[ 14%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_019.c.o
[ 14%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_021.c.o
[ 15%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_022.c.o
[ 15%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_023.c.o
[ 15%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_024.c.o
[ 15%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_025.c.o
[ 15%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_026.c.o
[ 15%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_027.c.o
[ 16%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_028.c.o
[ 16%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_029.c.o
[ 16%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_031.c.o
[ 16%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_030.c.o
[ 16%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_032.c.o
[ 16%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_033.c.o
[ 16%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_034.c.o
[ 17%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_036.c.o
[ 17%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_037.c.o
[ 17%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_038.c.o
[ 17%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_042.c.o
[ 17%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_041.c.o
[ 17%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_040.c.o
[ 18%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_043.c.o
[ 18%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_044.c.o
[ 18%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_045.c.o
[ 18%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_049.c.o
[ 18%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_047.c.o
[ 18%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_048.c.o
[ 18%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_046.c.o
[ 19%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_050.c.o
[ 19%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_051.c.o
[ 19%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_052.c.o
[ 19%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_053.c.o
[ 19%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_055.c.o
[ 19%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_056.c.o
[ 19%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_059.c.o
[ 20%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_057.c.o
[ 20%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_058.c.o
[ 20%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_060.c.o
[ 20%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_061.c.o
[ 20%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_062.c.o
[ 21%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_063.c.o
[ 21%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_064.c.o
[ 21%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_065.c.o
[ 21%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_066.c.o
[ 21%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_067.c.o
[ 21%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_068.c.o
[ 21%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_069.c.o
[ 21%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_071.c.o
[ 22%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_070.c.o
[ 22%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_072.c.o
[ 22%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_073.c.o
[ 22%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_074.c.o
[ 22%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_075.c.o
[ 23%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_076.c.o
[ 23%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_077.c.o
[ 23%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_078.c.o
[ 23%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_079.c.o
[ 23%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_080.c.o
[ 23%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_081.c.o
[ 23%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_082.c.o
[ 25%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_083.c.o
[ 25%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_085.c.o
[ 25%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_086.c.o
[ 25%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_088.c.o
[ 25%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_087.c.o
[ 26%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_091.c.o
[ 26%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_089.c.o
[ 26%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_094.c.o
[ 26%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_095.c.o
[ 26%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_093.c.o
[ 26%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_096.c.o
[ 26%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_097.c.o
[ 27%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_099.c.o
[ 27%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_100.c.o
[ 27%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_101.c.o
[ 27%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_103.c.o
[ 27%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_105.c.o
[ 27%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_104.c.o
[ 27%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_106.c.o
[ 28%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_107.c.o
[ 28%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_108.c.o
[ 28%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_112.c.o
[ 28%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_113.c.o
[ 28%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_111.c.o
[ 28%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_114.c.o
[ 29%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_115.c.o
[ 29%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_117.c.o
[ 29%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_116.c.o
[ 29%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_118.c.o
[ 29%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_119.c.o
[ 29%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_120.c.o
[ 30%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_123.c.o
[ 30%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_121.c.o
[ 30%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_125.c.o
[ 30%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_126.c.o
[ 30%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_132.c.o
[ 30%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_133.c.o
[ 30%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_134.c.o
[ 31%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_136.c.o
[ 31%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_137.c.o
[ 31%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_138.c.o
[ 31%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_139.c.o
[ 31%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_140.c.o
[ 31%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_141.c.o
[ 32%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_142.c.o
[ 32%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_143.c.o
[ 32%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_144.c.o
[ 32%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_145.c.o
[ 32%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_147.c.o
[ 32%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_149.c.o
[ 32%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_148.c.o
[ 33%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_150.c.o
[ 33%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_152.c.o
[ 33%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_153.c.o
[ 33%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_154.c.o
[ 33%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_156.c.o
[ 33%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_157.c.o
[ 34%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_159.c.o
[ 34%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_162.c.o
[ 34%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_163.c.o
[ 34%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_165.c.o
[ 34%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_164.c.o
[ 34%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_167.c.o
[ 35%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_168.c.o
[ 35%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_166.c.o
[ 35%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_170.c.o
[ 35%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_171.c.o
[ 35%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_172.c.o
[ 35%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_175.c.o
[ 35%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_173.c.o
[ 36%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_176.c.o
[ 36%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_177.c.o
[ 36%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_178.c.o
[ 36%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_180.c.o
[ 36%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_183.c.o
[ 36%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_182.c.o
[ 37%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_184.c.o
[ 37%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_185.c.o
[ 37%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_186.c.o
[ 37%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_187.c.o
[ 37%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_188.c.o
[ 37%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_189.c.o
[ 37%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_190.c.o
[ 38%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_193.c.o
[ 38%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_191.c.o
[ 38%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_192.c.o
[ 38%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_194.c.o
[ 38%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_195.c.o
[ 38%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_196.c.o
[ 39%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_197.c.o
[ 39%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_198.c.o
[ 39%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_199.c.o
[ 39%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_200.c.o
[ 39%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_203.c.o
[ 39%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_201.c.o
[ 39%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_202.c.o
[ 40%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_204.c.o
[ 40%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_205.c.o
[ 40%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_206.c.o
[ 40%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_209.c.o
[ 40%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_208.c.o
[ 40%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_210.c.o
[ 40%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_214.c.o
[ 40%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_215.c.o
[ 41%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_212.c.o
[ 41%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_216.c.o
[ 41%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_217.c.o
[ 41%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_218.c.o
[ 41%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_221.c.o
[ 42%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_219.c.o
[ 42%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_222.c.o
[ 42%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_225.c.o
[ 42%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_227.c.o
[ 42%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_226.c.o
[ 42%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_228.c.o
[ 43%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_229.c.o
[ 43%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_231.c.o
[ 43%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_230.c.o
[ 43%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_232.c.o
[ 43%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_233.c.o
[ 43%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_234.c.o
[ 43%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_236.c.o
[ 43%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_237.c.o
[ 44%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_235.c.o
[ 44%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_240.c.o
[ 44%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_242.c.o
[ 44%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_241.c.o
[ 44%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_238.c.o
[ 45%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_244.c.o
[ 45%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_245.c.o
[ 45%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_246.c.o
[ 45%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_249.c.o
[ 45%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_250.c.o
[ 45%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_252.c.o
[ 46%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_254.c.o
[ 46%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_256.c.o
[ 46%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_259.c.o
[ 46%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_261.c.o
[ 46%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_260.c.o
[ 46%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_262.c.o
[ 47%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_263.c.o
[ 47%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_266.c.o
[ 47%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_265.c.o
[ 47%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_267.c.o
[ 47%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_268.c.o
[ 47%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_269.c.o
[ 47%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_271.c.o
[ 47%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_274.c.o
[ 48%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_272.c.o
[ 48%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_281.c.o
[ 48%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_282.c.o
[ 48%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_284.c.o
[ 48%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_283.c.o
[ 48%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_286.c.o
[ 50%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_285.c.o
[ 50%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_287.c.o
[ 50%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_288.c.o
[ 50%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_289.c.o
[ 50%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_290.c.o
[ 50%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_291.c.o
[ 51%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_292.c.o
[ 51%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_295.c.o
[ 51%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_297.c.o
[ 51%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_298.c.o
[ 51%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_299.c.o
[ 51%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_300.c.o
[ 52%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_301.c.o
[ 52%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_302.c.o
[ 52%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_303.c.o
[ 52%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_305.c.o
[ 52%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_304.c.o
[ 52%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_306.c.o
[ 53%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_307.c.o
[ 53%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_309.c.o
[ 53%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_308.c.o
[ 53%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_311.c.o
[ 53%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_312.c.o
[ 54%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_315.c.o
[ 54%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_313.c.o
[ 54%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_314.c.o
[ 54%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_319.c.o
[ 54%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_320.c.o
[ 54%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_322.c.o
[ 54%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_323.c.o
[ 54%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_324.c.o
[ 54%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_328.c.o
[ 55%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_325.c.o
[ 55%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_327.c.o
[ 55%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_329.c.o
[ 55%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_331.c.o
[ 55%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_332.c.o
[ 55%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_333.c.o
[ 56%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_334.c.o
[ 56%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_335.c.o
[ 56%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_336.c.o
[ 56%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_337.c.o
[ 56%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_338.c.o
[ 56%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_339.c.o
[ 57%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_340.c.o
[ 57%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_343.c.o
[ 57%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_342.c.o
[ 57%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_341.c.o
[ 57%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_344.c.o
[ 57%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_345.c.o
[ 58%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_346.c.o
[ 58%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_347.c.o
[ 58%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_348.c.o
[ 58%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_349.c.o
[ 58%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_351.c.o
[ 58%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_350.c.o
[ 58%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_352.c.o
[ 58%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_355.c.o
[ 58%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_356.c.o
[ 58%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_357.c.o
[ 58%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_354.c.o
[ 59%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_353.c.o
[ 59%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_358.c.o
[ 59%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_360.c.o
[ 60%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_359.c.o
[ 60%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_362.c.o
[ 60%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_361.c.o
[ 60%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_368.c.o
[ 60%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_366.c.o
[ 60%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_369.c.o
[ 61%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_370.c.o
[ 61%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_372.c.o
[ 61%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_374.c.o
[ 61%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_375.c.o
[ 61%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_377.c.o
[ 61%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_380.c.o
[ 62%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_381.c.o
[ 62%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_382.c.o
[ 62%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_384.c.o
[ 62%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_387.c.o
[ 62%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_388.c.o
[ 62%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_386.c.o
[ 63%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_390.c.o
[ 63%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_389.c.o
[ 63%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_393.c.o
[ 63%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_394.c.o
[ 63%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_396.c.o
[ 63%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_395.c.o
[ 63%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_397.c.o
[ 63%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_400.c.o
[ 63%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_399.c.o
[ 64%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_398.c.o
[ 64%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_403.c.o
[ 64%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_401.c.o
[ 64%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_404.c.o
[ 65%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_406.c.o
[ 65%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_409.c.o
[ 65%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_410.c.o
[ 65%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_412.c.o
[ 65%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_411.c.o
[ 65%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_413.c.o
[ 65%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_414.c.o
[ 65%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_416.c.o
[ 66%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_415.c.o
[ 66%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_417.c.o
[ 66%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_420.c.o
[ 66%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_421.c.o
[ 66%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_428.c.o
[ 67%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_429.c.o
[ 67%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_431.c.o
[ 67%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_432.c.o
[ 67%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_433.c.o
[ 67%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_436.c.o
[ 67%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_434.c.o
[ 68%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_437.c.o
[ 68%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_438.c.o
[ 68%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_442.c.o
[ 68%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_446.c.o
[ 68%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_447.c.o
[ 68%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_451.c.o
[ 68%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_452.c.o
[ 69%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_455.c.o
[ 69%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_456.c.o
[ 69%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_457.c.o
[ 69%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_471.c.o
[ 69%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_512.c.o
[ 69%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_513.c.o
[ 69%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_518.c.o
[ 70%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_516.c.o
[ 70%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_517.c.o
[ 70%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_519.c.o
[ 70%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_522.c.o
[ 70%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_521.c.o
[ 71%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_525.c.o
[ 71%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_524.c.o
[ 71%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_526.c.o
[ 71%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_527.c.o
[ 71%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_528.c.o
[ 71%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_530.c.o
[ 71%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_529.c.o
[ 72%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_532.c.o
[ 72%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_538.c.o
[ 72%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_543.c.o
[ 72%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_539.c.o
[ 72%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_541.c.o
[ 72%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_547.c.o
[ 73%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_550.c.o
[ 73%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_554.c.o
[ 73%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_555.c.o
[ 73%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_556.c.o
[ 73%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_557.c.o
[ 73%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_558.c.o
[ 73%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_559.c.o
[ 75%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_560.c.o
[ 75%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_FDS.c.o
[ 75%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_561.c.o
[ 75%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_GameGenie.c.o
[ 75%] Building C object src/CMakeFiles/punes.dir/core/mappers/mapper_NSF.c.o
[ 75%] Building CXX object src/CMakeFiles/punes.dir/core/mappers/butterworth.cpp.o
[ 76%] Building CXX object src/CMakeFiles/punes.dir/core/mappers/butterworth_interface.cpp.o
[ 76%] Building C object src/CMakeFiles/punes.dir/core/mappers/EE93Cx6.c.o
[ 76%] Building C object src/CMakeFiles/punes.dir/core/mappers/FCG.c.o
[ 76%] Building C object src/CMakeFiles/punes.dir/core/mappers/FME7.c.o
[ 76%] Building CXX object src/CMakeFiles/punes.dir/core/mappers/hc55516.cpp.o
[ 76%] Building CXX object src/CMakeFiles/punes.dir/core/mappers/hc55516_interface.cpp.o
[ 76%] Building C object src/CMakeFiles/punes.dir/core/mappers/JV001.c.o
[ 77%] Building C object src/CMakeFiles/punes.dir/core/mappers/JYASIC.c.o
[ 77%] Building C object src/CMakeFiles/punes.dir/core/mappers/KS202.c.o
[ 77%] Building C object src/CMakeFiles/punes.dir/core/mappers/LZ93D50.c.o
[ 77%] Building C object src/CMakeFiles/punes.dir/core/mappers/MMC1.c.o
[ 77%] Building C object src/CMakeFiles/punes.dir/core/mappers/MMC2.c.o
[ 77%] Building C object src/CMakeFiles/punes.dir/core/mappers/MMC3.c.o
[ 78%] Building C object src/CMakeFiles/punes.dir/core/mappers/MMC4.c.o
[ 78%] Building C object src/CMakeFiles/punes.dir/core/mappers/N118.c.o
[ 78%] Building C object src/CMakeFiles/punes.dir/core/mappers/OneBus.c.o
[ 78%] Building CXX object src/CMakeFiles/punes.dir/core/mappers/serial_devices.cpp.o
[ 78%] Building C object src/CMakeFiles/punes.dir/extra/emu2413/emu2413.c.o
[ 78%] Building CXX object src/CMakeFiles/punes.dir/core/mappers/serial_devices_interface.cpp.o
[ 79%] Building C object src/CMakeFiles/punes.dir/core/mappers/SST39SF040.c.o
[ 79%] Building C object src/CMakeFiles/punes.dir/core/mappers/TXC.c.o
[ 79%] Building CXX object src/CMakeFiles/punes.dir/core/mappers/upd7756.cpp.o
[ 79%] Building C object src/CMakeFiles/punes.dir/core/mappers/VRC6.c.o
[ 79%] Building CXX object src/CMakeFiles/punes.dir/core/mappers/upd7756_interface.cpp.o
[ 79%] Building C object src/CMakeFiles/punes.dir/core/mappers/VRC2and4.c.o
[ 79%] Building C object src/CMakeFiles/punes.dir/core/mappers/VRC7.c.o
[ 80%] Building C object src/CMakeFiles/punes.dir/core/mappers/VRC7_snd.c.o
[ 80%] Building CXX object src/CMakeFiles/punes.dir/core/mappers/waveFile.cpp.o
[ 80%] Building CXX object src/CMakeFiles/punes.dir/core/mappers/wave_file_interface.cpp.o
[ 80%] Building C object src/CMakeFiles/punes.dir/core/apu.c.o
[ 80%] Building C object src/CMakeFiles/punes.dir/core/bck_states.c.o
[ 80%] Building C object src/CMakeFiles/punes.dir/core/cheat.c.o
[ 81%] Building C object src/CMakeFiles/punes.dir/core/cpu.c.o
[ 81%] Building C object src/CMakeFiles/punes.dir/core/detach_barcode.c.o
[ 81%] Building C object src/CMakeFiles/punes.dir/core/emu.c.o
[ 81%] Building C object src/CMakeFiles/punes.dir/core/emu_thread.c.o
[ 81%] Building C object src/CMakeFiles/punes.dir/core/external_calls.c.o
[ 81%] Building C object src/CMakeFiles/punes.dir/core/fds.c.o
[ 81%] Building C object src/CMakeFiles/punes.dir/core/fps.c.o
[ 82%] Building C object src/CMakeFiles/punes.dir/core/ines.c.o
[ 82%] Building C object src/CMakeFiles/punes.dir/core/input.c.o
[ 82%] Building C object src/CMakeFiles/punes.dir/core/irqA12.c.o
[ 82%] Building C object src/CMakeFiles/punes.dir/core/irql2f.c.o
[ 82%] Building C object src/CMakeFiles/punes.dir/core/jstick.c.o
[ 82%] Building C object src/CMakeFiles/punes.dir/core/main.c.o
[ 83%] Building C object src/CMakeFiles/punes.dir/core/mappers.c.o
[ 83%] Building C object src/CMakeFiles/punes.dir/core/nes.c.o
[ 83%] Building C object src/CMakeFiles/punes.dir/core/nsf.c.o
[ 83%] Building C object src/CMakeFiles/punes.dir/core/nsfe.c.o
[ 83%] Building C object src/CMakeFiles/punes.dir/core/overscan.c.o
[ 83%] Building C object src/CMakeFiles/punes.dir/core/palette.c.o
[ 84%] Building C object src/CMakeFiles/punes.dir/core/patcher.c.o
[ 84%] Building C object src/CMakeFiles/punes.dir/core/patcher_xdelta3_wrap.c.o
[ 84%] Building C object src/CMakeFiles/punes.dir/core/ppu.c.o
[ 84%] Building C object src/CMakeFiles/punes.dir/core/recording.c.o
[ 84%] Building C object src/CMakeFiles/punes.dir/core/rewind.c.o
[ 84%] Building C object src/CMakeFiles/punes.dir/core/rom_mem.c.o
[ 84%] Building C object src/CMakeFiles/punes.dir/core/save_slot.c.o
[ 85%] Building C object src/CMakeFiles/punes.dir/core/sha1.c.o
[ 85%] Building C object src/CMakeFiles/punes.dir/core/tape_data_recorder.c.o
[ 85%] Building C object src/CMakeFiles/punes.dir/core/uncompress.c.o
[ 85%] Building C object src/CMakeFiles/punes.dir/core/unif.c.o
[ 85%] Building C object src/CMakeFiles/punes.dir/core/vector.c.o
[ 85%] Building C object src/CMakeFiles/punes.dir/core/memmap.c.o
[ 86%] Building CXX object src/CMakeFiles/punes.dir/c++/crc/crc.cpp.o
/build/source/src/core/recording.c: In function 'recording_init':
/build/source/src/core/recording.c:207:41: error: implicit declaration of function 'avcodec_close'; did you mean 'avio_close'? [-Wimplicit-function-declaration]
  207 |                                         avcodec_close(test);
      |                                         ^~~~~~~~~~~~~
      |                                         avio_close
/build/source/src/core/recording.c: In function 'ffmpeg_video_add_stream_format_h264':
/build/source/src/core/recording.c:1057:40: error: 'FF_PROFILE_H264_HIGH' undeclared (first use in this function); did you mean 'AV_PROFILE_H264_HIGH'?
 1057 |                 video->avcc->profile = FF_PROFILE_H264_HIGH;
      |                                        ^~~~~~~~~~~~~~~~~~~~
      |                                        AV_PROFILE_H264_HIGH
/build/source/src/core/recording.c:1057:40: note: each undeclared identifier is reported only once for each function it appears in
/build/source/src/core/recording.c: In function 'ffmpeg_audio_select_sample_fmt':
/build/source/src/core/recording.c:1456:20: warning: 'sample_fmts' is deprecated [-Wdeprecated-declarations]
 1456 |         const enum AVSampleFormat *p = codec->sample_fmts;
      |                    ^~~~~~~~~~~~~~
In file included from /nix/store/c2dqa8h34ablc8nm7gxlhs4y0b8m50jj-ffmpeg-8.0-dev/include/libavcodec/avcodec.h:41,
                 from /build/source/src/core/recording.c:19:
/nix/store/c2dqa8h34ablc8nm7gxlhs4y0b8m50jj-ffmpeg-8.0-dev/include/libavcodec/codec.h:204:32: note: declared here
  204 |     const enum AVSampleFormat *sample_fmts; ///< @deprecated use avcodec_get_supported_config()
      |                                ^~~~~~~~~~~
/build/source/src/core/recording.c:1468:9: warning: 'sample_fmts' is deprecated [-Wdeprecated-declarations]
 1468 |         return (codec->sample_fmts[0]);
      |         ^~~~~~
/nix/store/c2dqa8h34ablc8nm7gxlhs4y0b8m50jj-ffmpeg-8.0-dev/include/libavcodec/codec.h:204:32: note: declared here
  204 |     const enum AVSampleFormat *sample_fmts; ///< @deprecated use avcodec_get_supported_config()
      |                                ^~~~~~~~~~~
/build/source/src/core/recording.c: In function 'ffmpeg_audio_select_samplerate':
/build/source/src/core/recording.c:1474:9: warning: 'supported_samplerates' is deprecated [-Wdeprecated-declarations]
 1474 |         if (!codec->supported_samplerates) {
      |         ^~
/nix/store/c2dqa8h34ablc8nm7gxlhs4y0b8m50jj-ffmpeg-8.0-dev/include/libavcodec/codec.h:202:16: note: declared here
  202 |     const int *supported_samplerates;       ///< @deprecated use avcodec_get_supported_config()
      |                ^~~~~~~~~~~~~~~~~~~~~
/build/source/src/core/recording.c:1491:9: warning: 'supported_samplerates' is deprecated [-Wdeprecated-declarations]
 1491 |         p = codec->supported_samplerates;
      |         ^
/nix/store/c2dqa8h34ablc8nm7gxlhs4y0b8m50jj-ffmpeg-8.0-dev/include/libavcodec/codec.h:202:16: note: declared here
  202 |     const int *supported_samplerates;       ///< @deprecated use avcodec_get_supported_config()
      |                ^~~~~~~~~~~~~~~~~~~~~
/build/source/src/core/recording.c:1497:9: warning: 'supported_samplerates' is deprecated [-Wdeprecated-declarations]
 1497 |         p = codec->supported_samplerates;
      |         ^
/nix/store/c2dqa8h34ablc8nm7gxlhs4y0b8m50jj-ffmpeg-8.0-dev/include/libavcodec/codec.h:202:16: note: declared here
  202 |     const int *supported_samplerates;       ///< @deprecated use avcodec_get_supported_config()
      |                ^~~~~~~~~~~~~~~~~~~~~
/build/source/src/core/recording.c: In function 'ffmpeg_audio_select_channel_layout':
/build/source/src/core/recording.c:1611:9: warning: 'ch_layouts' is deprecated [-Wdeprecated-declarations]
 1611 |         if (!codec->ch_layouts) {
      |         ^~
/nix/store/c2dqa8h34ablc8nm7gxlhs4y0b8m50jj-ffmpeg-8.0-dev/include/libavcodec/codec.h:226:28: note: declared here
  226 |     const AVChannelLayout *ch_layouts;
      |                            ^~~~~~~~~~
/build/source/src/core/recording.c:1631:9: warning: 'ch_layouts' is deprecated [-Wdeprecated-declarations]
 1631 |         p = codec->ch_layouts;
      |         ^
/nix/store/c2dqa8h34ablc8nm7gxlhs4y0b8m50jj-ffmpeg-8.0-dev/include/libavcodec/codec.h:226:28: note: declared here
  226 |     const AVChannelLayout *ch_layouts;
      |                            ^~~~~~~~~~
/build/source/src/core/recording.c:1642:9: warning: 'ch_layouts' is deprecated [-Wdeprecated-declarations]
 1642 |         p = codec->ch_layouts;
      |         ^
/nix/store/c2dqa8h34ablc8nm7gxlhs4y0b8m50jj-ffmpeg-8.0-dev/include/libavcodec/codec.h:226:28: note: declared here
  226 |     const AVChannelLayout *ch_layouts;
      |                            ^~~~~~~~~~
make[2]: *** [src/CMakeFiles/punes.dir/build.make:7046: src/CMakeFiles/punes.dir/core/recording.c.o] Error 1
make[2]: *** Waiting for unfinished jobs....
make[1]: *** [CMakeFiles/Makefile2:212: src/CMakeFiles/punes.dir/all] Error 2
make: *** [Makefile:136: all] Error 2
```

### Additional context

_No response_

### System metadata

 - system: `"x86_64-linux"`
 - host os: `Linux 6.12.57-xanmod1, NixOS, 25.11 (Xantusia), 25.11.20251108.b6a8526`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.32.3`
 - nixpkgs: `/nix/store/js3h59i67va69xbf90pdvcabd0ddqkcg-source`

### Notify maintainers

@OPNA2608 

---

**Note for maintainers:** Please tag this issue in your pull request description. (i.e. `Resolves #ISSUE`.)


### I assert that this issue is relevant for Nixpkgs

- [x] I assert that this is a bug and not a support request.
- [x] I assert that this is not a [duplicate of an existing issue](https://github.com/NixOS/nixpkgs/issues?q=is%3Aopen+is%3Aissue+label%3A%220.kind%3A+build+failure%22). 
- [x] I assert that I have read the [NixOS Code of Conduct](https://github.com/NixOS/.github/blob/master/CODE_OF_CONDUCT.md) and agree to abide by it.

### Is this issue important to you?

Add a :+1: [reaction] to [issues you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[issues you find important]: https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc

--- a/pkgs/by-name/pu/punes/package.nix
+++ b/pkgs/by-name/pu/punes/package.nix
@@ -47,7 +47,22 @@ stdenv.mkDerivation (finalAttrs: {
       url = "https://github.com/punesemu/puNES/commit/ff906e0a79eeac9a2d16783e0accf65748bb275e.patch";
       hash = "sha256-+s7AdaUBgCseQs6Mxat/cDmQ77s6K6J0fUfyihP82jM=";
     })
+
+    # Fix compat with FFmpeg 8, part 1
+    # Remove when version > 0.111
+    (fetchpatch {
+      name = "0003-punes-Dont-use-deprecated-avcodec_close.patch";
+      url = "https://github.com/punesemu/puNES/commit/49f86fcf0fab37d4761b713b0a9e7dc342b8f594.patch";
+      hash = "sha256-l71n+c7W2vJKOJKLgfMfDJtof+UBzcVUgXMTn6ap1XI=";
+    })
   ];
+
+  # Fix compat with FFmpeg 8, part 2
+  # Remove when https://github.com/punesemu/puNES/pull/444 merged & in release
+  postPatch = ''
+    substituteInPlace src/core/recording.c \
+      --replace-fail 'FF_PROFILE_H264_HIGH' 'AV_PROFILE_H264_HIGH'
+  '';
 
   nativeBuildInputs = [
     cmake

